tests/monotouch-test/ModelIO/MDLObject.cs

```diff
@@ -63,6 +63,19 @@ public void GetBoundingBox ()
 				var bb = obj.GetBoundingBox (0);
 			}
 		}
+
+		[Test]
+		public void IndexerTest ()
+		{
+			using (var obj = new MDLObject ()) {
+				var key = new Protocol ("MDLComponent");
```

Comment:
I think just a `[Wrapper]` that calls a conversion method (or a new Protocol ctor) would be fine.

BTW your example is wrong :smile: It should be:

> var key = new Protocol (typeof (IMDLComponent));